### PR TITLE
fixed S3 bucket options merge error

### PR DESCRIPTION
**Pull request by superchell**
in block in `objectsaws-sdk-s3 (1.30.1) lib/aws-sdk-s3/bucket.rb` I got error `undefined method merge for nil:NilClass`
because the objects method in bucket.rb waiting a hash:
`   def objects(options = {})
      batches = Enumerator.new do |y|
        options = options.merge(bucket: @name)`

### CODE REVIEW

File: camaleon_cms.gemspec [hunk 1]
```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'will_paginate-bootstrap'
   s.add_dependency 'breadcrumbs_on_rails'
   s.add_dependency 'font-awesome-rails'
-  s.add_dependency 'tinymce-rails', '~> 4.3'
+  s.add_dependency 'tinymce-rails', '~> 4.2.5'
```
Comment (owen2345) [inline camaleon_cms.gemspec]: @superchell why this change?

Comment (superchell) [reply camaleon_cms.gemspec]: @owen2345 sorry, this commit shouldn't have gotten into this PR

Comment (owen2345) [reply camaleon_cms.gemspec]: @superchell 
I understand, please update the pull request to approve it.

Comment (superchell) [reply camaleon_cms.gemspec]: @owen2345 done!
